pkg/gce-pd-csi-driver/controller.go

```diff
@@ -1561,6 +1566,9 @@ func (gceCS *GCEControllerServer) CreateSnapshot(ctx context.Context, req *csi.C
 			return nil, err
 		}
 	case common.DiskImageType:
+		if isHdHA {
```

Comment:
Looks like this also isn't supported for regional pd: https://docs.google.com/document/d/1cJOw1tbD6JB6e3ePeskgNdZxRG2gQGHC-6aZ_x1qBpM/edit?disco=AAABAIXTNuM. See: https://cloud.google.com/compute/docs/disks/regional-persistent-disk#restrictions

Please add that as well. Looks like we missed it first time around